exwm-background.el

```diff
@@ -0,0 +1,201 @@
+;;; exwm-background.el --- X Background Module for EXWM  -*- lexical-binding: t -*-
+
+;; Copyright (C) 2022 Free Software Foundation, Inc.
+
+;; Author: Steven Allen <steven@stebalien.com>
+
+;; This file is part of GNU Emacs.
```

Comment:
(I've signed the copyright assignment paperwork)

Reply:
That's wonderful news :-).  Thank you.